### PR TITLE
implemented github action workflow for process paid fillings

### PR DESCRIPTION
**Pull request by hanlunBCRegistries**
Description:

* implemented workflow for process paid fillings

### CODE REVIEW

File: jobs/process_paid_filings/devops/vaults.gcp.env [hunk 1]
```diff
@@ -0,0 +1,18 @@
+DATABASE_USERNAME="op://database/$APP_ENV/business-ar-db/DATABASE_USERNAME"
+DATABASE_PASSWORD="op://database/$APP_ENV/business-ar-db/DATABASE_PASSWORD"
+DATABASE_NAME="op://database/$APP_ENV/business-ar-db/DATABASE_NAME"
+DATABASE_UNIX_SOCKET="op://database/$APP_ENV/business-ar-db/DATABASE_UNIX_SOCKET"
```
Comment (bolyachevets) [inline jobs/process_paid_filings/devops/vaults.gcp.env:4]: DATABASE_UNIX_SOCKET  is not used in the codebase. You probably need to copy this from AR Reminder job: https://github.com/bcgov/business-ar/blob/a67f02e3dd49feeb5e3457f3f068208c72f04aa1/jobs/ar-reminder/src/ar_reminder/config.py#L72

Comment (bolyachevets) [reply jobs/process_paid_filings/devops/vaults.gcp.env:4]: Probably a good idea to check config.py to see which variables are read there, so that there is 1-to-1 corresponding between this files and read variables.

File: jobs/process_paid_filings/Dockerfile [hunk 2]
```diff
@@ -79,12 +81,14 @@ RUN --mount=type=cache,target="$POETRY_CACHE_DIR" \
...
-USER web
-
-# The following stage is only for production:
-FROM development_build AS production_build
-COPY --chown=web:web . /code
-
-# ENV PYTHONPATH=/opt/app-root/src
-
-CMD [ "python", "-m", "process_paid_filings" ]
+  COPY --chown=web:web . /code
```
Comment (bolyachevets) [inline jobs/process_paid_filings/Dockerfile:84]: why did you change indentation?

Comment (hanlunBCRegistries) [reply jobs/process_paid_filings/Dockerfile]: fixed it, copied from the ar-reminder dockerfile and didn't realize the indentation was changed

File: jobs/process_paid_filings/devops/vaults.gcp.env (hunk 1 above)
Comment (hanlunBCRegistries) [reply jobs/process_paid_filings/devops/vaults.gcp.env:4]: I think this job didn't really establish connection to the database, probably I should just delete those lines in vaults.gcp.env?

**Review by bolyachevets — COMMENTED** [review]
need to regenerate poetry.lock and commit the updated file

**Review by bolyachevets — APPROVED** [review]

File: jobs/process_paid_filings/src/process_paid_filings/config.py [hunk 3]
```diff
@@ -81,7 +81,10 @@ class TestConfig(_Config):  # pylint: disable=too-few-public-methods
     DATABASE_TEST_HOST = os.getenv("DATABASE_TEST_HOST", "")
     DATABASE_TEST_PORT = int(os.getenv("DATABASE_TEST_PORT", "5432"))  # POSTGRESQL
 
-    SQLALCHEMY_DATABASE_URI = f"postgresql://{DATABASE_TEST_USERNAME}:{DATABASE_TEST_PASSWORD}@{DATABASE_TEST_HOST}:{DATABASE_TEST_PORT}/{DATABASE_TEST_NAME}"
+    SQLALCHEMY_DATABASE_URI = (
```
Comment (bolyachevets) [inline jobs/process_paid_filings/src/process_paid_filings/config.py:84]: also delete this

File: jobs/process_paid_filings/devops/gcp/clouddeploy.yaml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+   profiles: [dev]
+   strategy:
+    standard:
+      verify: false
+   deployParameters:
+   - values:
+      deploy-env: "development"
+      deploy-project-id: "a083gt-dev"
+      job-name: "business-ar-process-paid-filings-dev"
+      cloudsql-instances: "a083gt-dev:northamerica-northeast1:businesses-db-dev"
```
Comment (bolyachevets) [inline jobs/process_paid_filings/devops/gcp/clouddeploy.yaml:46]: leave empty string here, if db is not used

File: jobs/process_paid_filings/devops/gcp/clouddeploy.yaml [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+   profiles: [test]
+   strategy:
+    standard:
+      verify: false
+   deployParameters:
+   - values:
+      deploy-env: "development"
+      deploy-project-id: "a083gt-test"
+      job-name: "business-ar-process-paid-filings-test"
+      cloudsql-instances: "a083gt-test:northamerica-northeast1:businesses-db-test"
```
Comment (bolyachevets) [inline jobs/process_paid_filings/devops/gcp/clouddeploy.yaml:59]: leave empty string here, if db is not used

File: jobs/process_paid_filings/devops/gcp/clouddeploy.yaml [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+   profiles: [sandbox]
+   strategy:
+    standard:
+      verify: false
+   deployParameters:
+   - values:
+      deploy-env: "sandbox"
+      deploy-project-id: "a083gt-integration"
+      job-name: "business-ar-process-paid-filings-sandbox"
+      cloudsql-instances: "a083gt-integration:northamerica-northeast1:businesses-db-sandbox"
```
Comment (bolyachevets) [inline jobs/process_paid_filings/devops/gcp/clouddeploy.yaml:72]: leave empty string here, if db is not used

File: jobs/process_paid_filings/devops/gcp/clouddeploy.yaml [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+   profiles: [prod]
+   strategy:
+    standard:
+      verify: false
+   deployParameters:
+   - values:
+      deploy-env: "production"
+      deploy-project-id: "a083gt-prod"
+      job-name: "business-ar-process-paid-filings-prod"
+      cloudsql-instances: "a083gt-prod:northamerica-northeast1:businesses-db-prod"
```
Comment (bolyachevets) [inline jobs/process_paid_filings/devops/gcp/clouddeploy.yaml:85]: leave empty string here, if db is not used

File: jobs/process_paid_filings/src/process_paid_filings/config.py [hunk 8]
```diff
@@ -81,7 +81,10 @@ class TestConfig(_Config):  # pylint: disable=too-few-public-methods
     DATABASE_TEST_HOST = os.getenv("DATABASE_TEST_HOST", "")
```
Comment (bolyachevets) [inline jobs/process_paid_filings/src/process_paid_filings/config.py:81]: remove DATABASE testing vars

**Review by bolyachevets — APPROVED** [review]

**Comment by bolyachevets** [comment]
fix linting

**Review by bolyachevets — APPROVED** [review]
